bridge_ui/src/components/Recovery.tsx

```diff
@@ -335,6 +335,10 @@ export default function Recovery() {
               targetAddress: parsedPayload.targetAddress,
               originChain: parsedPayload.originChain,
               originAddress: parsedPayload.originAddress,
+              amount:
```

Comment:
For some context - this could also be a NFT payload which doesn't have an amount property, so I have to do this to get around Typescript complaining